templates/express/package.json

```diff
@@ -4,7 +4,7 @@
   "scripts": {
     "build": "remix build",
     "dev": "remix build && run-p \"dev:*\"",
-    "dev:node": "cross-env NODE_ENV=development nodemon ./server.js --watch ./server.js",
+    "dev:node": "cross-env NODE_ENV=development nodemon --require dotenv/config ./server.js --watch ./server.js",
```

Comment:
If we depend on `dotenv`, it should be explicitly listed in our `devDependencies`